hack/manifests/csi/vsphere/01-example-driver-daemonset.yaml

```diff
@@ -45,7 +45,7 @@ spec:
                 - --mode=kubelet-registration-probe
             initialDelaySeconds: 3
         - name: vsphere-csi-node
-          image: gcr.io/cloud-provider-vsphere/csi/release/driver:v3.0.0
+          image: registry.k8s.io/csi-vsphere/driver:v3.3.0
```

Comment:
please call out that this change is proposing a new repo for vSphere CSI driver image **and bumping the driver version**

Reply:
sure, I've done so in the PR description